src/views/Game/index.js

```diff
@@ -23,22 +23,20 @@ class Game extends Component {
   };
 
   render() {
-    const {
-      players, deck, screenplay, killPlayer
-    } = this.props;
-    const playersList = Object.keys(deck).map(p => (
+    const { game, screenplay, killPlayer } = this.props;
+    const playersList = Object.keys(game).map(p => (
       <ListItem avatar key={p}>
         <Left>
           <Thumbnail source={require('../../assets/full-moon.jpg')} />
         </Left>
         <Body>
-          <Text style={styles.name}>{players[p].name}</Text>
-          <Text style={styles.role} note>
-            {deck[p].card.role}
+          <Text style={game[p].alive ? styles.nameAlive : styles.nameDead}>{game[p].name}</Text>
+          <Text style={game[p].alive ? styles.roleAlive : styles.roleDead} note>
+            {game[p].card.role}
           </Text>
         </Body>
         <Right>
-          <Button bordered danger onPress={() => killPlayer(p)}>
+          <Button bordered={game[p].alive} danger onPress={() => killPlayer(p)}>
```

Comment:
Shouldn't this button be disabled after the player has been killed?

Reply:
I agree with you.